Resources/doc/reference/installation.rst

```diff
@@ -25,7 +25,7 @@ Retrieve the bundle with composer:
 
 .. code-block:: bash
 
-    $ php composer.phar require sonata-project/media-bundle
+    $ composer require sonata-project/media-bundle
```

Comment:
The `$` sign was already there, so let's say it is not in the scope of this PR, shall we?